docker/env.example.sepolia

```diff
@@ -0,0 +1,17 @@
+# source rpc url for local deployment
+FORK_RPC_URL=https://ethereum-sepolia-rpc.publicnode.com
+
+DEPLOY_ENV=LOCAL
+ETHERSCAN_API_KEY=
+LOCAL_ETHEREUM_RPC_URL=http://localhost:8545
+TESTNET_RPC_URL=
+
+# FIXME: place your project's metadata URI here
+METADATA_URI=https://wavs.xyz/metadata.json
```

Comment:
Yeah, this is the one I meant about metadata uri. There is nothing on this link.
See #200 for the follow up